src/App.js

```diff
@@ -3,13 +3,25 @@ import logo from "./logo.png";
 import HomePage from "./pages/home-page/HomePage";
 import VideoPage from "./pages/video-page/VideoPage";
 import { Route, Routes } from "react-router-dom";
+import WatchLater from "./pages/watch-later-page/WatchLater";
+import LoginPage from "./pages/login-page/LoginPage";
+import SignupPage from "./pages/signup-page/SignupPage";
+import HistoryPage from "./pages/history-page/HistoryPage";
+import PlaylistListing from "./pages/playlist-listing-page/PlaylistListing";
+import CreatePlaylist from "./pages/create-playlist-page/CreatePlaylist";
```

Comment:
Create one index file. use Barrel import

Reply:
yes will try in next merge